airflow/models/baseoperator.py

```diff
@@ -1745,16 +1661,17 @@ def from_operator(cls, operator: BaseOperator, mapped_kwargs: Dict[str, Any]) ->
         return MappedOperator(
             operator_class=type(operator),
             task_id=operator.task_id,
-            task_group=getattr(operator, 'task_group', None),
-            dag=getattr(operator, '_dag', None),
+            task_group=operator.task_group,
+            dag=dag,
+            upstream_task_ids=operator.upstream_task_ids,
+            downstream_task_ids=operator.downstream_task_ids,
             start_date=operator.start_date,
             end_date=operator.end_date,
             partial_kwargs={k: v for k, v in operator_init_kwargs.items() if k != "task_id"},
             mapped_kwargs=mapped_kwargs,
             owner=operator.owner,
             max_active_tis_per_dag=operator.max_active_tis_per_dag,
             deps=operator.deps,
-            params=operator.params,
```

Comment:
Why was params removed?

Reply:
It’s now included in `partial_kwargs`, same as `pool`, `queue`, etc.